doc/whats_new/v0.21.rst

```diff
@@ -26,6 +26,14 @@ Changelog
   (regression introduced in 0.21) :issue:`13910`
   by :user:`Jérémie du Boisberranger <jeremiedbb>`.
 
+:mod:`sklearn.utils.sparsefuncs`
```

Comment:
Better off putting it under preprocessing?

Reply:
Could do, although the code change is in sparsefuncs and will also affect `feature_selection.VarianceThreshold` once pull #13704 goes through.

Reply:
+1, I believe that in this case the two classes that are potentially impacted by this bug are: `LabelBinarizer` and `MaxAbsScaler`.

Reply:
I had not read the previous comment when writing mine. In the end I am fine with the current PR.